crates/forge/tests/it/zk/gas.rs

```diff
@@ -79,10 +79,39 @@ forgetest_async!(zk_script_execution_with_gas_per_pubdata, |prj, cmd| {
     let private_key = get_rich_wallet_key();
 
     // Test with unacceptable gas per pubdata (should fail)
-    let zero_pubdata_args = create_script_args(&private_key, &url, "--zk-gas-per-pubdata", "1");
-    cmd.arg("script").args(&zero_pubdata_args);
-    cmd.assert_failure();
-    cmd.forge_fuse();
+    let forge_bin = env!("CARGO_BIN_EXE_forge");
```

Comment:
I think you can retrieve the right binary with `prj.forge_bin()`... Technically you can even retrieve the command with `cmd.cmd()` but I think you won't be able to use it later since it's just a `&mut Command`

Reply:
Will do!